csrc/python_frontend/python_bindings.cpp

```diff
@@ -488,6 +488,80 @@ void initNvFuserPythonBindings(PyObject* module) {
         return ss.str();
       });
 
+  //! KernelProfiles are encapsulated in FusionProfiles where each KP
+  //! is associated with a segment.
+  py::class_<KernelProfile> kernel_prof(nvfuser, "KernelProfile");
```

Comment:
Can we also had the scheduler heuristic used per segment?

Reply:
Yes, it will be a separate PR to add a new field.